src/common/transformations/include/ov_ops/type_relaxed.hpp

```diff
@@ -95,7 +97,7 @@ class OPENVINO_API TypeRelaxedBase {
     } init_rt_result;
 
     init_rt_result init_rt_info(Node& node) const {
-        node.get_rt_info()["opset"] = "type_relaxed_opset";
+        node.get_rt_info()["opset"] = std::string(version_prefix) + node.get_type_info().version_id;
```

Comment:
Previous hardcoded value ("type_relaxed_opset") is still used in 
https://github.com/openvinotoolkit/openvino/blob/4aed649c1cc8cef6c105e8076e5852c8e51ce944/src/common/transformations/include/transformations/utils/print_model.hpp#L325-L327

Could you please update the check? Maybe it is safer to use dynamic cast and check on nullptr instead of rt info check

Reply:
Updated